js/src/frameworks/langgraph.ts

```diff
@@ -0,0 +1,32 @@
+import { LangchainToolSet as BaseComposioToolSet } from "./langchain";
+import { COMPOSIO_BASE_URL } from "../sdk/client/core/OpenAPI";
+import type { Optional } from "../sdk/types";
+import { WorkspaceConfig } from "../env/config";
+import { Workspace } from "../env";
+
+export class LanggraphToolSet extends BaseComposioToolSet {
```

Comment:
We should use cameCase here like, "LangGraph" - they also use similar format in their docs.
<img width="339" alt="Screenshot 2024-12-05 at 1 30 37 PM" src="https://github.com/user-attachments/assets/b3c12f85-4161-4ef0-8cf3-f5343130794f">